frontend/src/components/atoms/Table/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  useReactTable,
+  Column,
+  TableOptions,
+  RowData,
+} from '@tanstack/react-table';
+import Input from '../inputs/Input';
+import TablePagination from '../TablePagination';
+import './style.css';
+
+declare module '@tanstack/table-core' {
```

Comment:
I do not undestand this, can you explain this to me?

Reply:
It's a way to declare what data the Column's `meta` property is able to receive. Here's the doc : https://tanstack.com/table/v8/docs/api/core/table#meta